File: sandbox.client/front/ts/model/ClientRecord.ts
export class ClientRecord {
  public id: number;
  public name: string;
  public surname: string;
  public patronymic: string;
  public age: number;
  public middle_balance: number | null;
  public max_balance: number | null;
  public min_balance: number | null;

  public assign(o: ClientRecord): ClientRecord {
    this.id = o.id;
    this.name = o.name;
    this.surname = o.surname;
    this.patronymic = o.patronymic;
    this.age = o.age;
    this.middle_balance = o.middle_balance;
    this.max_balance = o.max_balance;
    this.min_balance = o.min_balance;
    return this;
  }

  public static copy(a: ClientRecord): ClientRecord {
    let ret = new ClientRecord();
    ret.assign(a);
    return ret;
  }
}

File: sandbox.controller/src/kz/greetgo/sandbox/controller/model/ClientRecord.java
package kz.greetgo.sandbox.controller.model;

public class ClientRecord {
  public Integer id;
  public String name;
  public String surname;
  public String patronymic;
  public int age;
  public float middle_balance;
  public float max_balance;
  public float min_balance;

  @Override
  public String toString() {
    return "ClientRecord{" +
      "id=" + id +
      ", name='" + name + '\'' +
      ", surname='" + surname + '\'' +
      ", patronymic='" + patronymic + '\'' +
      ", age=" + age +
      ", middle_balance=" + middle_balance +
      ", max_balance=" + max_balance +
      ", min_balance=" + min_balance +
      '}';
  }
}
